.github/workflows/stale.yml

```diff
@@ -0,0 +1,57 @@
+name: 'Close stale issues and PRs'
+
+on:
+  schedule:
+    # Daily at 1:30 AM UTC
+    - cron: '30 1 * * *'
+
+env:
+  LC_ALL: en_US.UTF-8
+
+defaults:
+  run:
+    shell: bash
+
+permissions:
+  contents: read
+
+jobs:
+  stale:
+    permissions:
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Stale Action"
+        uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e # v9.0.0
```

Comment:
It's generally considered safe to use the release tag for first party actions, what do you think about changing this to `actions/stale@v9`?

Reply:
I agree that it's safe. I guess I prefer it for the sake of consistency since not everyone editing these files would know the reasoning behind when you would or wouldn't use this style.

I don't feel that strongly about it, though.

related PR: https://github.com/vllm-project/vllm/pull/9430

Reply:
I don't feel that strongly about it either, and it gets updated by dependabot anyway (as indicated by the other PR you referenced) then there's no harm in going with security best practice.

Reply:
The one slight downside is that the `v9` reference will just get moved as minor releases get made. With this style, we'll get dependabot PRs for each minor update. It's a bit more PR noise, but it's also kind of nice to be able to track exactly when versions change in case problems pop up.